app/Helpers/database/user.php

```diff
@@ -327,7 +328,7 @@ function GetDeveloperStatsFull(int $count, int $offset = 0, int $sortBy = 0, int
     $query = "SELECT ua.ID, COUNT(*) AS OpenTickets
               FROM Ticket tick
               INNER JOIN Achievements ach ON ach.ID=tick.AchievementID
-              INNER JOIN UserAccounts ua ON ua.User=ach.Author
+              INNER JOIN UserAccounts ua ON ua.ID = ach.user_id
```

Comment:
I think this `ua` could be eliminated too.

Reply:
Yep, good call. Done in latest